stages/org.osbuild.ostree.aleph

```diff
@@ -0,0 +1,165 @@
+
+    aleph_version_data = {
+        "osbuild-version": osbuild.__version__,
+        "version": aleph_version,
+        "ref": aleph_ref,
+        "ostree-commit": aleph_ostree_commit
+    }
+
+    return aleph_version_data
+
+
+def aleph_container(tree, imgref):
+    # extract the image name from the imgref
+    imgref_list = imgref.split(':')
+    if imgref_list[0] in ["ostree-remote-registry", "ostree-remote-image"]:
+        img_name = ':'.join(imgref_list[2:])
+    elif imgref_list[0] in ["ostree-image-signed", "ostree-unverified-registry"]:
+        img_name = ':'.join(imgref_list[1:])
+    else:
+        raise ValueError("Could not find 'ostree-remote-registry', \
+                         'ostree-remote-image', 'ostree-image-signed', or \
+                         'ostree-unverified-registry' in imgref")
+
+    img_name = img_name.lstrip('docker://')
+
+    extra_args = []
+    extra_args.append(f"--repo={tree}/ostree/repo")
+    extra_args.append(f"registry:{img_name}")
+
+    container_data_json = ostree.cli("container", "image", "metadata", *extra_args).stdout.rstrip()
+    container_data = json.loads(container_data_json)
+
+    extra_args.append("--config")
+    container_data_config_json = ostree.cli("container", "image", "metadata", *extra_args).stdout.rstrip()
+    container_data_config = json.loads(container_data_config_json)
+
+    aleph_digest = container_data['config']['digest']
+    aleph_ref = f"docker://{imgref}"
+    aleph_version = container_data_config['config']['Labels']['org.opencontainers.image.version']
+    aleph_container_image = container_data_config['config']['Labels']
+
+    aleph_version_data = {
+        "osbuild-version": osbuild.__version__,
+        "ref": aleph_ref,
+        "version": aleph_version,
+        "container-image": {
+            "image-name": img_name,
+            "image-digest": aleph_digest,
+            "image-labels": aleph_container_image
+        }
+    }
+    
+    # the 'ostree.commit' label will be optional in the future so
+    # prevent hard failing if key is not found
+    aleph_ostree_commit = container_data_config['config']['Labels'].get('ostree.commit')
+    if aleph_ostree_commit is not None:
+        aleph_version_data["ostree-commit"] = aleph_ostree_commit
+
+    return aleph_version_data
+
+
+def construct_aleph_json(tree, origin):
+    deploy_type, imgref = ostree.parse_origin(origin)
+    data = {}
+    # null deploy_type and imgref error is caught in the parse_origin() function
+    if deploy_type == "container":
+        data = aleph_container(tree, imgref)
+    elif deploy_type == "ostree_commit":
+        data = aleph_commit(tree, imgref)
+    
+    return data
+
+
+def main(tree, options):
+    coreos_compat = options.get("coreos_compat", False)
+    dep = options.get("deployment", None)
+    osname = ""
+    ref = ""
+    serial = None
+    origin = ""
+    aleph_filename = ".aleph-version.json"
+    coreos_aleph_filename = ".coreos-aleph-version.json"
```

Comment:
I meant at the file level, near e.g. `CAPABILITIES`.

Reply:
Made the following change:
```
...
CAPABILITIES = ["CAP_MAC_ADMIN"]
ALEPH_FILENAME = ".aleph-version.json"
COREOS_ALEPH_FILENAME = ".coreos-aleph-version.json"
...
```
Is this what you meant?